beacon-chain/rpc/eth/beacon/handlers_test.go

```diff
@@ -3874,3 +3874,16 @@ func TestServer_broadcastBlobSidecars(t *testing.T) {
 	require.NoError(t, server.broadcastSeenBlockSidecars(context.Background(), blk, b.GetDeneb().Blobs, b.GetDeneb().KzgProofs))
 	require.LogsContain(t, hook, "Broadcasted blob sidecar for already seen block")
 }
+
+func TestPublishBlobs(t *testing.T) {
+	server := &Server{
+		BlobReceiver:  &chainMock.ChainService{},
+		Broadcaster:   &mockp2p.MockBroadcaster{},
+		SyncChecker:   &mockSync.Sync{IsSyncing: false},
+	}
+	request := httptest.NewRequest(http.MethodPost, "http://foo.example", bytes.NewReader([]byte(rpctesting.PublishBlobsRequest)))
+	writer := httptest.NewRecorder()
+	writer.Body = &bytes.Buffer{}
+	server.PublishBlobs(writer, request)
+	assert.Equal(t, http.StatusOK, writer.Code)
+}
```

Comment:
This is a really basic test. A handler that does nothing would pass the test.

```go
func (s *Server) PublishBlobs(w http.ResponseWriter, r *http.Request) {
  // Nothing here and the test still passes!
}
```

Please add some more to this test to ensure that the method is doing what you expect. At a minimum, this should be asserting that the blobs were broadcasted (happy path). It's probably not necessary to test all of the unhappy paths (like returning an error when syncing).

Reply:
Updated